sdk/iot/hub/inc/az_iot_hub_client.h

```diff
@@ -453,41 +421,16 @@ AZ_NODISCARD az_result az_iot_hub_client_methods_response_get_publish_topic(
  */
 
 /**
- * @brief Gets the MQTT topic filter to subscribe to twin operation responses.
- *
- * @param[in] client The #az_iot_hub_client to use for this call.
- * @param[out] mqtt_topic_filter A buffer with sufficient capacity to hold the MQTT topic filter.
- *                               If successful, contains a null-terminated string with the topic
- *                               filter that needs to be passed to the MQTT client.
- * @param[in] mqtt_topic_filter_size The size, in bytes of \p mqtt_topic_filter.
- * @param[out] out_mqtt_topic_filter_length __[nullable]__ Contains the string length, in bytes, of
- *                                                         \p mqtt_topic_filter. Can be `NULL`.
- * @return #az_result
+ * @brief The MQTT topic filter to subscribe to twin operation responses.
+ * @remark Twin MQTT Publish messages will have QoS At Most Once (0).
  */
-AZ_NODISCARD az_result az_iot_hub_client_twin_response_get_subscribe_topic_filter(
-    az_iot_hub_client const* client,
-    char* mqtt_topic_filter,
-    size_t mqtt_topic_filter_size,
-    size_t* out_mqtt_topic_filter_length);
+#define AZ_IOT_HUB_CLIENT_TWIN_RESPONSE_SUBSCRIBE_TOPIC "$iothub/twin/res/#"
 
 /**
  * @brief Gets the MQTT topic filter to subscribe to twin desired property changes.
- * @remark The payload will contain only changes made to the desired properties.
- *
- * @param[in] client The #az_iot_hub_client to use for this call.
- * @param[out] mqtt_topic_filter A buffer with sufficient capacity to hold the MQTT topic filter.
- *                               If successful, contains a null-terminated string with the topic
- *                               filter that needs to be passed to the MQTT client.
- * @param[in] mqtt_topic_filter_size The size, in bytes of \p mqtt_topic_filter.
- * @param[out] out_mqtt_topic_filter_length __[nullable]__ Contains the string length, in bytes, of
- *                                                         \p mqtt_topic_filter. Can be `NULL`.
- * @return #az_result
+ * @remark Twin MQTT Publish messages will have QoS At Most Once (0).

```

Comment:
I am a bit confused by the capitalization choice here. Is `At Most Once` a domain specific term and what is `(0)` referring to?

Reply:
@ahsonkhan : MQTT has 3 Quality of Service (QoS) levels. Read about them here: https://www.hivemq.com/blog/mqtt-essentials-part-6-mqtt-quality-of-service-levels/. They are numbered 0 (At most once), 1 (at least once), & 2 (exactly once)

Reply:
Checked with the MQTT spec. I'll change the capitalization to At most once (0), etc